Tahi-project/tahi
# APERTA-5828: Remove s from questions_attachments misspelling

Fix for misspelling causing file download error. 

Related to: https://github.com/Tahi-project/tahi/pull/2016

JIRA issue: https://developer.plos.org/jira/browse/APERTA-5828
#### How to test
- Run the app ; create a paper; go to the Additional Information card
- Choose "Yes" on the first question (e..g "Have the results.... been published elsewhere?") 
- Upload a file ; after it uploads you will see the file name as a link to view the contents
- Refresh ; verify your uploaded file still shows up under the question
- Click file name to download the file that you just uploaded (i.e. "print+for+B.sketch" which was the name of the file that was uploaded)

![image](https://cloud.githubusercontent.com/assets/82035/12590172/91b40ea8-c42f-11e5-8c6c-19cec65ec476.png)
